src/app/services/favorite.service.ts

```diff
@@ -28,20 +30,35 @@ export class FavoriteService {
   }
 
   insert(custId: number, prodId: number): Observable<any> {
-    return this.http.post(`${environment.apiBase}/favorite`,
-    {userId: custId, productId: prodId})
+    let fave = new Favorite();
+    let cust = new User();
+    let product = new Product();
+    cust.userId = custId;
+    product.productId = prodId;
+    fave.user = cust;
+    fave.product = product;
+    return this.http.post(`${environment.apiBase}/favorite`, fave)
     .pipe( map( (response: any) => response));
   }
 
   update(favoriteId: number, custId: number, prodId: number): Observable<any> {
-    return this.http.put(`${environment.apiBase}/favorite`,
-    {favoriteId: favoriteId, userId: custId, productId: prodId})
+    let fave = new Favorite();
+    let cust = new User();
+    let product = new Product();
+    fave.favoriteId = favoriteId;
+    cust.userId = custId;
+    product.productId = prodId;
+    fave.user = cust;
+    fave.product = product;
+    return this.http.put(`${environment.apiBase}/favorite`, fave)
     .pipe( map( (response: any) => response));
   }
 
   delete(favoriteId: number): Observable<any> {
+    let fave = new Favorite();
+    fave.favoriteId = favoriteId;
     return this.http.delete(`${environment.apiBase}/favorite?favoriteId=${favoriteId}`)
```

Comment:
will not work, send a favorite object

Reply:
can i get the .delete in a slack message